src/components/NormalizationDashboard.jsx

```diff
@@ -9,7 +9,8 @@ function NormalizationDashboard() {
       <div className="mx-[139px]">
         <StatusBarsForNormalization />
       <div className="mt-[40px]">
-        <GlobalSearch searchString={"Search"} className="bg-white w-full h-[64px] items-center pl-[24px] text-sm placeholder-gray-20 rounded border border-grey-30 shadow" />
+        <GlobalSearch searchString={"Search"} inputClasses="bg-white w-full h-[64px] items-center pl-[24px] text-sm placeholder-gray-20 rounded border border-grey-30 shadow"
```

Comment:
Either enclose the classes in {} or dont.
Pick one standard and follow it through out the repo.
Dont mix and match both.

Reply:
done